fbc/Makefile

```diff
@@ -61,60 +46,56 @@ fbc-onboarding: clean
 			--cache-dir /workspace/.catalog_cache
 
 .PHONY: catalogs
-# replace this stub with one customized to serve your needs ... some examples below
-
-# here are a few examples of different approaches to fulfilling this target
-# comment out / customize the one that makes the most sense, or use them as examples in defining your own
-#
-# --- BASIC TEMPLATE ---
-catalogs: basic
-#
-# --- SEMVER TEMPLATE ---
-#catalogs: semver
-
-
-# basic target provides an example FBC generation from a `basic` template type.
-# this example takes a single file as input and generates a well-formed FBC operator contribution as an output
-.PHONY: basic
-basic: ${BINDIR}/opm clean
-	for version in $(OCP_VERSIONS); do \
-		mkdir -p ${CATALOG_DIR}/$${version}/${OPERATOR_NAME}/ && \
-		${BINDIR}/opm alpha render-template basic -o yaml ${OPERATOR_CATALOG_TEMPLATE_DIR}/$${version}.yaml > ${CATALOG_DIR}/$${version}/${OPERATOR_NAME}/catalog.yaml; \
-	done
-
-
-# semver target provides an example FBC generation from a `semver` template type.
-# this example takes a single file as input and generates a well-formed FBC operator contribution as an output
-.PHONY: semver
-semver: ${BINDIR}/opm clean
-	for version in $(OCP_VERSIONS); do \
-		mkdir -p ${CATALOG_DIR}/$${version}/${OPERATOR_NAME}/ && \
-		${BINDIR}/opm alpha render-template semver -o yaml  ${OPERATOR_CATALOG_TEMPLATE_DIR}/$${version}.yaml > ${CATALOG_DIR}/$${version}/${OPERATOR_NAME}/catalog.yaml; \
-	done
+
+# Catalogs are generated based on the mapping in ci.yaml file
+# The example of the mapping is:
+# fbc:
+#   enabled: true
+#   catalog_mapping:
+#     - template_name: basic.yaml
+#       catalog_names: ["v4.14", "v4.15", "v4.16"]
+#       type: olm.template.basic
+#     - template_name: semver.yaml
+#       catalog_names: ["v4.13", "v4.17"]
+#       type: olm.semver
+catalogs: ${BINDIR}/render_catalogs.sh ${BINDIR}/opm ${BINDIR}/yq clean
+	@echo "Rendering catalogs from templates"
+	@${BINDIR}/render_catalogs.sh
 
 
 # validate-catalogs target illustrates FBC validation
 # all FBC must pass opm validation in order to be able to be used in a catalog
 .PHONY: validate-catalogs
 validate-catalogs: ${BINDIR}/opm
-	for version in $(OCP_VERSIONS); do \
-		${BINDIR}/opm validate $(CATALOG_DIR)/$${version}/${OPERATOR_NAME} && echo "$${version} catalog validation passed" || echo "$${version} catalog validation failed"; \
-	done
+	@find ${TOPDIR}/catalogs -type d -name fbc-test-operator -exec \
+		sh -c '${BINDIR}/opm validate $$(dirname "{}") && echo "✅ Catalog validation passed: {}" || echo "❌ Catalog validation failed: {}"' \;
```

Comment:
This no longer works, it's basically doing a no-op output.

Reply:
Why is that? When I make a mistake in the catalog on purpose this gives me the error and the command fails.

Reply:
I had to merge this to not break users with incompatible Makefiles. Let's still discuss this and if we find it is not correct I am going to fix it in next PR.